flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
:-(

Reply:
> :-(

Yes I have removed the i386 tarball. I thought that was what you mean in this comment https://github.com/flathub/flathub/pull/2684#discussion_r768831424. Unfortunately this is an Electron app. Do you think it can be released for other architectures like ARM? If so I'll be happy to propose that upstream.

Reply:
Oh I see that there's an ARM zip here: https://github.com/pilas-engine/pilas-engine/releases . So I'll add that one. Should I add `pilas-engine-linux-32_bits.zip` too? If so is `i386` the right arch? I'm looking at this post for reference: https://blogs.gnome.org/tvb/2016/06/24/flatpak-builds-available-on-a-variety-of-architectures/

Reply:
no it's the lack of aarch64 I'm sad.

Reply:
both `i386` and `arm` support have been discontinued for at least a couple of years now. Looking forward.

Reply:
OK, so removing that last fixup commit then.

Reply:
> no it's the lack of aarch64 I'm sad.

Good point. I have added an issue upstream https://github.com/pilas-engine/pilas-engine/issues/40, it should be easy to add aarch64 since electron-packager supports it.